src/main/webapp/index.jsp

```diff
@@ -0,0 +1,92 @@
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<% boolean isUserLoggedIn = (boolean) request.getAttribute("isUserLoggedIn"); %>
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>CodeU Starter Project</title>
+    <link rel="stylesheet" href="/css/main.css">
+  </head>
+  <body>
```

Comment:
You need to add "background-color: #8cdbe2;" to main.css if you want to maintain the lovely blue background color you had.